### ISSUE TITLE
Use mockoon as upload server for testing

### ISSUE THREAD

**Issue by clabnet** [mockoon/mockoon#1445]
Hi, 
can be useful use Mockoon as server to upload static files (e.g., images) to test upload development code and related options.
The `static` folder should be the same to serve the uploaded files.

**Comment by 255kb** [comment]
I think adding static file serving is a bit outside of an API mocking tool's scope. 
However, I'm currently improving the support for multipart/form-data files, which was inexistent until now.
Aside from normal multipart fields, files uploaded will get described in the parsed request body, including their size, file name and type:
```json
{
  field1: "value",
  files: [
    {
      fieldname: "file",
      filename: "test.json",
      mimetype: "application/json",
      size: 1024      
    }
  ]
}
```

Combined with a global variable or the new setData helper, and this [file serving technique](https://mockoon.com/tutorials/create-endpoint-serving-static-file/), it should be enough to simulate a small static server if needed.

**Comment by 255kb** [comment]
📦This was released in [v9.0.0](https://mockoon.com/releases/9.0.0/)
